apps/apps.h

```diff
@@ -568,7 +568,7 @@ void store_setup_crl_download(X509_STORE *st);
 
 # define APP_PASS_LEN    1024
 
-# define SERIAL_RAND_BITS        64
+# define SERIAL_RAND_BITS        (18 * 8)
```

Comment:
Can't be 20 since that could encode with a leading zero and RFC 5280 says no more than 20 bytes.  Could be 19; do you want that?